File: ACL/frontend/ACLProject/src/app/filepermission.ts
export class FilePermission {
    public u_id: number;
    public file_id: number;
    public p_type: string;
    constructor(u_id: number,file_id:number,p_type: string)
    {
    this.u_id=u_id;
    this.file_id=file_id;
    this.p_type=p_type;
    }
 }

File: ACL/backend/rest-api/model/filepermission.go
package model

type FilePermission struct {
	Id            int64  `json:"u_id,string,omitempty" key:"primary" column:"u_id"`
	FId           int64  `json:"file_id,string,omitempty" column:"file_id"`
	PType         string `json:"p_type" column:"p_type"`
}

func (filepermission *FilePermission) Table() string {
	return "file_permission"
}


func (filepermission *FilePermission) String() string {
	return Stringify(filepermission)
}
